src/App.js

```diff
@@ -2,24 +2,59 @@ import React, { Component } from 'react';
 import './App.css';
 import Header from './components/Header';
 import Footer from './components/Footer';
+import Chart from './components/Chart';
+import Rank from './components/Rank';
 
 
 class App extends Component {
-  render() {
+
+    constructor() {
+        super();
+        this.state={
+            cDate: {}
+        }
+    }
+
+    componentWillMount() {
```

Comment:
Let's change this to componentDidMount.

For this reason: https://stackoverflow.com/questions/46043832/why-componentwillmount-should-not-be-used

Reply:
will change that